config/routes.rb

```diff
@@ -246,7 +246,11 @@
 
     resources :permissions, :only => [:index]
 
-    resources :auth_source_ldaps, :except => [:show]
+    resources :auth_source_ldaps, :except => [:show] do
+      collection do
+        put 'test_connection'
```

Comment:
I didn't realize this could be used for models not already saved in the db - with credentials already saved in Foreman. My bad.